### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ricardo123321**
in this code we made a bubble sort method and a bubble sort-by method who organize the elements given in an array or a block

### CODE REVIEW

File: feature.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  loop do
+    moved = false
+    (array.length - 1).times do |i|
+      if array[i] > array[i + 1]
+        array[i], array[i + 1] = array[i + 1], array[i]
+        moved = true
+      end
+    end
+    break unless moved
+  end
```
Comment (shshamim63) [inline feature.rb:11]: bubble sort returns a sorted array. But, based on your current formation it returns nil. after line 11, place `array`.

File: feature.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    moved = false
+    (array.length - 1).times do |i|
+      block_comparison = yield(array[i], array[i + 1])
+      if block_comparison.positive?
+        array[i], array[i + 1] = array[i + 1], array[i]
+        moved = true
+      end
+    end
+    break unless moved
+  end
```
Comment (shshamim63) [inline feature.rb:26]: Same in here too. it is returning nil. So after line 25 place `array`

**Review by shshamim63 — CHANGES_REQUESTED** [review]
## Status :traffic_light: Required Changes :tractor: 
___
Hi @ricardo123321 and @paulo-techie :wave: ,
Awesome job. Just a couple of things to do before declaring this project as approved. Check the comments below...
___
### :spades: Mandatory Changes :spades: 
___
1. Both of your methods are not returning the modified array. Please check the comments below to fix them.
2. I can not see any descriptive readme in this repository. Please add one.
___
##### Please fix the above-mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @ricardo123321 and @paulo-techie :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewer, your methods are working fine, please merge this branch and continue congratulations :tada: :tada: :tada: 

[OPTIONAL]

- [x] I strongly recommend the use of `array.size` instead of `array.length` to keep the Ruby convention

PROJECT STATUS: APPROVED :green_apple:

**Pull request merged**